index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>UI Challenge 2</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" media="screen" href="css/base.css" />
+  <link rel="stylesheet" type="text/css" media="screen" href="css/layout.css" />
+  <link rel="stylesheet" type="text/css" media="screen" href="css/modules.css" />
+  
+</head>
+<body>
+  <h1>UI Challenge- Custom Form Elements</h1>
+  <h2>text input</h2>
+  <input type="text" placeholder="placeholder text">
+
+  <h2>text input w/ validation error</h2>
+  <input type="text" placeholder="placeholder text">

```

Comment:
I wish you'd added the validation error here, probably using the `invalid` pseudoselector.